.github/workflows/change.yaml

```diff
@@ -0,0 +1,45 @@
+name: Check for API breaks
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  check:
+    runs-on: ubuntu-latest
+    permissions:
+      pull-requests: write
```

Comment:
Just as an FYI - last I tried this, it didn't work when the PR comes from a fork.